library/HTMLPurifier/HTML5Config.php

```diff
@@ -77,6 +80,14 @@ public function __construct(HTMLPurifier_ConfigSchema $schema, HTMLPurifier_Prop
             $schema->add('HTML.Forms', false, 'bool', false);
         }
 
+        if (empty($schema->info['HTML.Link'])) {
```

Comment:
Rename setting to `HTML.SafeLink`. See https://github.com/xemlock/htmlpurifier-html5/pull/63/files#r634757873

Reply:
Will rename as part of https://github.com/xemlock/htmlpurifier-html5/pull/63#discussion_r634757873